multichain-testing/scripts/update-config.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+## see https://github.com/cosmology-tech/starship/blob/1d60f55c631b4d0f92a43ad92e9a935298aa3aa5/starship/charts/devnet/scripts/default/update-config.sh
+
+CHAIN_ID="${CHAIN_ID:=osmosis}"
+CHAIN_DIR="${CHAIN_DIR:=$HOME/.osmosisd}"
+KEYS_CONFIG="${KEYS_CONFIG:=configs/keys.json}"
+
+set -eux
+
+ls $CHAIN_DIR
+
+
+echo "Update config.toml file"
+sed -i -e 's#"tcp://127.0.0.1:26657"#"tcp://0.0.0.0:26657"#g' $CHAIN_DIR/config/config.toml
+sed -i -e 's/index_all_keys = false/index_all_keys = true/g' $CHAIN_DIR/config/config.toml
+sed -i -e 's/seeds = ".*"/seeds = ""/g' $CHAIN_DIR/config/config.toml
+sed -i -e 's#cors_allowed_origins = \[\]#cors_allowed_origins = \["*"\]#g' $CHAIN_DIR/config/config.toml
+
+echo "Increase `*_bytes` parameters for MsgInstallBundle"
```

Comment:
please comment on where these values come from, with enough info that one can cross-check in case either side changes

Reply:
Agree. It seems we only have `RPC_MAX_BODY_BYTES` in [agoric-cli's chain-config.js](https://github.com/Agoric/agoric-sdk/blob/master/packages/agoric-cli/src/chain-config.js#L66), and nothing for `max_header_bytes` and `max_txs_bytes`. I transparently documented how these values were derived, and we can fix-forward if we learn there are better values to use.